lib/numbers_and_words/strategies/figures_converter/languages/nl.rb

```diff
@@ -10,16 +10,11 @@ def hundreds_number_to_words
           end
 
           def capacity_iteration
-            if FiguresArray::THOUSAND_CAPACITY == @current_capacity
-              if figures.number_in_capacity(@current_capacity) == 1
-                megs
-              else
-                capacity_words = words_in_capacity(@current_capacity)
-                capacity_words.empty? ? [] : [capacity_words, megs].join
-              end
-            else
-              super
-            end
+            return super if FiguresArray::THOUSAND_CAPACITY != @current_capacity
```

Comment:
I think this could be reformatted to make it clearer that the main choice is between thousands and not thousands. Maybe a blank line after this one if you want to avoid nesting completely?

Reply:
Or, leave the code structure as it was before this PR, and only use a guard clause inside the `if FiguresArray::THOUSAND_CAPACITY == @current_capacity` branch.

```ruby
            if FiguresArray::THOUSAND_CAPACITY == @current_capacity
              return megs if figures.number_in_capacity(@current_capacity) == 1

              capacity_words = words_in_capacity(@current_capacity)
              capacity_words.empty? ? [] : [capacity_words, megs].join
            else
              super
            end
```

Reply:
Sure, it's a personal preference.

Probably you'll be reviewing any addition to this code, so better fits your preferred style. Feel free to change it to your desired style (as long as it is OK with RuboCop, Coveralls and Code Climate).

Reply:
There are a few "best practices" or whatever, that RuboCop enforces now and then. Since it helps maintaining our code, we just... comply.

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause

Apparently in Ruby, it is considered a "better" style. I was just future proofing it.